README.md

```diff
@@ -96,6 +96,7 @@ See [server demo](example) and [browser demo](https://github.com/bcherny/json-sc
 | unreachableDefinitions | boolean | `false` | Generates code for `definitions` that aren't referenced by the schema. |
 | strictIndexSignatures | boolean | `false` | Append all index signatures with `\| undefined` so that they are strictly typed. |
 | $refOptions | object | `{}` | [$RefParser](https://github.com/BigstickCarpet/json-schema-ref-parser) Options, used when resolving `$ref`s |
+| booleanSchemas | boolean | false | Generate respectively `any` and `never` types for boolean schema values `true` and `false` |
```

Comment:
This behavior should be the default -- it doesn't need to be configurable. Let's remove this option.